storybook/src/Storybook/FigmaUtils.qml

```diff
@@ -0,0 +1,42 @@
+pragma Singleton
```

Comment:
Personally I like qml singletons for more consistent imports on client side. Importing js can be done in several ways, it easily leads to inconsistencies (https://doc.qt.io/qt-5/qtqml-javascript-imports.html). In qml singletons it's also easy to hide implementation details (e.g helper methods) using our `d` convention. 

If you know some advantages of js files over qml singletons I'm open for discussion :)

Btw QML stateful singetons are very bad idea and should be treated as a last resort. We have such things in the project and we suffer a lot bc of that :D
Js imports also can hold state, what of course is bad idea as well :)

Reply:
Regarding the import I don't see any difference. Js files can be added to qmldir and be imported as any other qml. See https://doc.qt.io/qt-6/qtqml-modules-qmldir.html#example-of-a-qmldir-file as an example.

The singleton approach is a bit heavier, but it's true. It offers the qml sweets (and bitterness because there are lots of bugs on qml singletons). There is a nice read here on how js functions will be resolved https://doc.qt.io/qt-6/qtqml-javascript-resources.html#shared-javascript-resources-libraries. Not much different than singletons, but without the qml engine, bindings, etc.

Reply:
Good point, I forget about that method of adding js files :). Hard to say what is better. For now I'm leaving it as it is.